infra/src/test/kotlin/fi/fta/geoviite/infra/publication/PublicationServiceIT.kt

```diff
@@ -1407,8 +1416,8 @@ constructor(
             }
         assertEquals(1, diff.size)
         assertEquals("description", diff[0].propKey.key.toString())
-        assertEquals(trackNumber.description, diff[0].value.oldValue)
-        assertEquals(updatedTrackNumber.description, diff[0].value.newValue)
+        assertEquals(trackNumber.description.toString(), diff[0].value.oldValue.toString())
+        assertEquals(updatedTrackNumber.description.toString(), diff[0].value.newValue.toString())
```

Comment:
Näitä `.toString()`:ejä joutui lisäilemään ärsyttävän paljon. Katson näitä jatkokehityksenä

Reply:
Hmm, niin miksikäs nuo joutuu lisäämään? Onko tuolla diffissä joku eri tyyppi kuin descriptionbase?

Reply:
Sitäpä juuri ajattelin koittaa selvitellä. FreeTextin kanssa näitä ei tarvinnut, joten pitäisi ottaa selvää mikä on näissä eri tavalla 🤔  Työlistalla siinä kohtaa kun pääsen review-korjauksiin saakka